.gitignore

```diff
@@ -22,3 +22,6 @@
 /build_artifacts
 
 *.pyc
+
+# Rattler-build's artifacts are in `output` when not specifying anything.
```

Comment:
Please revert this change for now. We should keep changes like those in their own PRs b/c they are not recipe/new package.

Reply:
The gitignore change? It was added automatically on re-render.

Reply:
Oh. Sorry, missed that :-/